packages/repository/test/acceptance/has-many.relation.acceptance.ts

```diff
@@ -59,6 +55,43 @@ describe('HasMany relation', () => {
     expect(persisted.toObject()).to.deepEqual(order.toObject());
   });
 
+  it('can patch many instances', async () => {
+    await createSamples();
+    await testPatch();
+
+    async function testPatch() {
```

Comment:
What is the purpose of nesting the test code inside `testPatch`? We usually inline the test code in the test function itself.

```ts
it('can patch many instances', async () => {
  await createSamples();
  const patchObject = {description: 'new order'};
  // etc.
});
```